fw/vhost.c

```diff
@@ -2141,7 +2197,7 @@ tfw_cfgop_frang_strict_host_checking(TfwCfgSpec *cs, TfwCfgEntry *ce)
 	int r;
 	FrangVhostCfg *cfg = tfw_cfgop_frang_get_cfg();
 
-	if (ce->dflt_value && cfg->http_strict_host_checking)
+	if (ce->dflt_value && !cfg->http_strict_host_checking)
```

Comment:
We need to discuss this place. If I understand it correctly we have the same problem for `http_body_len` also and may be some other settings.

Reply:
I returned the old behavior, only explicit parameters should be overridden.